release/release_tests.yaml

```diff
@@ -3360,18 +3350,21 @@
     wait_for_nodes:
       num_nodes: 6
 
+  smoke_test:
+    frequency: nightly
+
+    run:
+      timeout: 2000
+
   alert: default
 
   variations:
     - __suffix__: aws
-    - __suffix__: smoke-test
-      frequency: nightly
-      run:
-        timeout: 2000
-        script: python stress_tests/run_stress_tests.py
     - __suffix__: gce
       env: gce
       frequency: manual
+      smoke_test:
+        frequency: manual
```

Comment:
adding 

   run:
      timeout: 2000

I wish it can just inherit from the default value but currently you have to copy the whole thing here, due to my dummy use of dict merge